RecoTracker/LSTCore/src/alpaka/LSTEvent.dev.cc

```diff
@@ -433,9 +435,13 @@ void LSTEvent::createTriplets() {
 }
 
 void LSTEvent::createTrackCandidates(bool no_pls_dupclean, bool tc_pls_triplets) {
-  if (!trackCandidatesDC_) {
-    trackCandidatesDC_.emplace(n_max_nonpixel_track_candidates + n_max_pixel_track_candidates, queue_);
-    auto buf = trackCandidatesDC_->buffer();
+  if (!trackCandidatesBaseDC_) {
+    trackCandidatesBaseDC_.emplace(n_max_nonpixel_track_candidates + n_max_pixel_track_candidates, queue_);
+    auto buf = trackCandidatesBaseDC_->buffer();
+    alpaka::memset(queue_, buf, 0u);
```

Comment:
Could be
```suggestion
    trackCandidatesBaseDC_->zeroInitialise(queue_);
```
(also elsewhere this pattern is used)

Reply:
Thank you, Matti! I wasn't aware of this functionality.

Reply:
I made this change. Thanks!